51-android.rules

```diff
@@ -959,12 +958,9 @@ LABEL="adb", ENV{adb_adb}="yes", GOTO="android_usb_rule_match"
 
 # ADB Debug and Audio Source
 LABEL="adbaud", ENV{adb_adb}="yes"
-LABEL="aud", ENV{adb_user}="yes", GOTO="android_usb_rule_match"
 
 # ADB Debug and AT-commands CDC Serial
 LABEL="adbcdc", ENV{adb_adbcdc}="yes"
-# check if exists /dev/ttyACM%n, GROUP=dialout, modeprobe cdc_acm)
-LABEL="cdc", ENV{adb_user}="yes", GOTO="android_usb_rule_match"
```

Comment:
Same thing here. Since there is no code (above) making use of the label "cdc", then we keep the GOTO.
Just copy the GOTO to the line above. Then it is okay to delete both lines here.